chart/templates/rendered.yaml

```diff
@@ -812,6 +812,8 @@ spec:
         - --health-probe-bind-address=:8081
         - --metrics-bind-address=127.0.0.1:8080
         - --leader-elect
+        - --flagd-cpu-limit=0.5
```

Comment:
We should pass these values as value injections with helm, you can overwrite the values through the overlay in `/config/overlays/helm` with a `patchesStrategicMerge`. Which will let developers update these values via the `values.yaml` file.
Happy to introduce this change in a separate PR

Reply:
I will introduce this change in https://github.com/open-feature/open-feature-operator/issues/271